src/components/options/providers/contextMenuItems.svelte

```diff
@@ -86,24 +96,40 @@ initData();
           </div>
 
           <div class="flex-1">
-            <div class="d-flex align-items-center">
+            <div class="d-flex align-items-start">
               <div class="flex-1">
                 <input
                   type="text"
                   value="{item.label}"
                   class="form-control text-black"
+                  class:is-invalid="{!item.label &&
+                    !hideErrors[`${index}.label`]}"
                   placeholder="Label to be used in the context menu"
+                  on:input="{() => onInput(index, 'label')}"
+                  on:blur="{() => onBlur(index, 'label')}"
                   on:change="{(e) =>
                     onChange(index, 'label', e.target.value)}" />
+                <div class="invalid-feedback ml-1">
+                  Label should not be empty
+                </div>
               </div>
               <div class="flex-2 mx-2">
                 <input
                   type="text"
                   value="{item.link}"
                   class="form-control text-info"
+                  class:is-invalid="{!isUrl(item.link) &&
+                    !hideErrors[`${index}.link`]}"
                   placeholder="URL address to which send requests"
+                  on:input="{() => onInput(index, 'link')}"
+                  on:blur="{() => onBlur(index, 'link')}"
                   on:change="{(e) =>
                     onChange(index, 'link', e.target.value)}" />
+                <div class="invalid-feedback ml-1">
+                  {item.link
+                    ? "URL should be a valid URL"
```

Comment:
`URL should be a valid URL` don't feels good, maybe `This value isn't a valid URL` or `This value must be a valid URL`.